extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php

```diff
@@ -6,7 +6,12 @@ public function __construct( Array $data = array() ) {
 		$this->setData( $data );
 	}
 
-	public function enabled ( $wgCityId, $user ) {
+	public function enabled ( $wgCityId, User $user ) {
+		// disable if all Wikia email disabled
+		if ( $user->getBoolOption( "unsubscribed" ) ) {
```

Comment:
single quotes are preferred according to guidelines: http://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#String_literals